pyproject.toml

```diff
@@ -97,3 +92,15 @@ line-length = 100
 [tool.ruff.lint]
 select = ["E", "I", "F"]
 isort.order-by-type = false
+
+[tool.scikit-build]
+cmake.build-type = "Release"
+cmake.version = ">=3.15"
+# By default, sdist-build-core uses `.gitignore` to select what to exclude from the source
+# distribution. This list is to explict exclude files not specified in `.gitignore`.
+sdist.exclude = [
```

Comment:
I think it's because this exclude list applies to very repo (including submodules). I tried to change `cmake-build-*` to `cmake*`, then the `cmake` directory under clp/components/core would also be excluded.